### ISSUE TITLE
CSV import supports email address as identifier but Users list UI doesn't render it properly

### ISSUE THREAD

**Issue by intelliant01** [learningequality/kolibri#7461]
### Observed behavior

CSV import instructions at **`Facility`** - **`DATA`** - **`Import and export users`** - **`View spreadsheet format reference`** suggests using email address as an `Identifier` -

![image](https://user-images.githubusercontent.com/2577060/89799303-ec7e8980-db4a-11ea-8eed-5f84c7e5855c.png)

But an email address overlaps with gender field even for short emails in the  **`Facility`** - **`USERS`** list -

![image](https://user-images.githubusercontent.com/2577060/89799510-336c7f00-db4b-11ea-9c3a-598136de00de.png)

### Expected behavior

Identifier field's value should not overlap with gender field value.
 
### User-facing consequences

Unreadable data

### Errors and logs

…

### Steps to reproduce

…

### Context

Kolibri 0.14.0

**Comment by jredrejo** [comment]
@intelliant01 could you elaborate a bit more where is the issue? I don't quite understand it.  If you could attach a csv to test it, it would be great.

**Comment by jonboiser** [comment]
@jredrejo This isn't an issue with CSV import, it's just that the Identifier column on the 'Facility > User' field has a max-width of 120px, which isn't wide enough for long email addresses.

https://github.com/learningequality/kolibri/blob/0beb63c921f4568a228c03fb96f444319818016d/kolibri/plugins/facility/assets/src/views/UserTable.vue#L265-L268

**Comment by intelliant01** [comment]
@jredrejo the issue is not per se with the CSV import process. Just edit a user's profile and set an average length email address and choose a gender for any user and we have the email address and gender values overlapping. 

I am referring to the CSV import because it is only in those instructions that it suggests using an email id for the identifier field.

**Comment by pcenov** [comment]
@radinamatic this one seems to be resolved in 0.15.6 as a slider is displayed if the value is too long:

![2022-08-03_15-20-28](https://user-images.githubusercontent.com/79847249/182607132-2539039d-f053-4951-9d7b-96aea3811daa.png)

**Comment by rtibbles** [comment]
Closing as fixed.

**Comment by pcenov** [comment]
@rtibbles actually we were just discussing with Radina that this one is still not fixed in Firefox and IE so it should be reopened:

IE11:
![2022-08-03_18-15-39](https://user-images.githubusercontent.com/79847249/182645905-fc5b892a-4acd-45c5-ad03-b454fe156ded.png)
Firefox:
![2022-08-03_18-19-16](https://user-images.githubusercontent.com/79847249/182646100-900a77ed-2a7b-49e4-801c-18db3cb619fa.png)

**Comment by radinamatic** [comment]
Still a few bumps remain to flatten...

**Comment by rtibbles** [comment]
Ahah.

**Comment by pcenov** [comment]
@radinamatic this is still a valid issue for Firefox in 0.16.b10 but only when the email is long and the resolution is less than 900 px so it can probably be closed as a rare edge case with no significant impact.

**Comment by radinamatic** [comment]
I'm thinking this might be a valid test case for when we approach the feature of the responsive and accessible tables... 🤔 
cc @marcellamaki

**Comment by marcellamaki** [comment]
@radinamatic yes, good idea. it does seem like something that we could consider and test for that feature, but I'm not sure that keeping this issue in particular open will help me remember. Let me think about where we can document that so we remember!
